components/factories/AssessmentGrader/GradingForm/GradingForm.tsx

```diff
@@ -0,0 +1,63 @@
+import { InputField } from "@components/shared/forms/InputField";
+import styles from "./GradingForm.module.css";
+import React, { useCallback, useState } from "react";
+import { TextAreaField } from "@components/shared/forms/TextAreaField";
+import { Button } from "@components/shared/elements/Button";
+import { Loader } from "@components/shared/elements/Loader";
+
+interface Props {
+  grader: (grade: number, note: string) => Promise<void>;
+  status: "initial" | "loading" | "fetched" | "error"
+}
+
+const GradingForm = ({ grader, status } : Props) => {
+  
+  const [gradeInput, setGradeInput] = useState("");
+  const [notes, setNotes] = useState("");
+
+  const forceTextIntoNumber = (inputtedNumber: string) => {
```

Comment:
Hello Alex, may I know the purpose of this function? I'm curious to know why we must do this rather than just using a number input. Is this some sort of trick maybe or it's more efficient perhaps? Let me know :D

Reply:
Hello, Rashad! I believe in this case I was just reusing the component that we already have, which uses a text input. If necessary, I can rework the input field to force the numbers. I hope it explains! 😄

Reply:
Ahh I see Alex. Okay LGTM!